packages/flutter/lib/src/widgets/animated_cross_fade.dart

```diff
@@ -216,7 +216,6 @@ class AnimatedCrossFade extends StatefulWidget {
   /// [AnimatedCrossFadeBuilder].
   static Widget defaultLayoutBuilder(Widget topChild, Key topChildKey, Widget bottomChild, Key bottomChildKey) {
     return Stack(
-      overflow: Overflow.visible,
```

Comment:
if you change the default to hardEdge make sure to update this too

Reply:
Thanks for catching it! Done.